fastapi-python-web-apis/main.py

```diff
@@ -0,0 +1,174 @@
+import random
+from typing import Annotated
+
+from pydantic import BaseModel, Field
+
+from fastapi import FastAPI, HTTPException, Query
+from fastapi.middleware.cors import CORSMiddleware
+
+tags_metadata = [
+    {
+        "name": "Random Playground",
+        "description": "Generate random numbers",
+    },
+    {
+        "name": "Random Items Management",
+        "description": "Create, shuffle, read, update and delete items",
+    },
+]
+
+app = FastAPI(
+    title="Randomizer API",
+    description="Shuffle lists, pick random items, generate random numbers.",
+    version="1.0.0",
+    openapi_tags=tags_metadata,
+)
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["http://localhost:3000", "https://example.com"],
+    allow_credentials=True,
+    allow_methods=["GET", "POST", "PUT", "DELETE"],
+    allow_headers=["*"],
+)
+
+items_db = []
+
+
+class Item(BaseModel):
+    name: str = Field(
+        min_length=1, max_length=100, description="The item name"
+    )
+
+
+class ItemResponse(BaseModel):
+    message: str
+    item: str
+
+
+class ItemListResponse(BaseModel):
+    original_order: list[str]
+    randomized_order: list[str]
+    count: int
+
+
+class ItemUpdateResponse(BaseModel):
+    message: str
+    old_item: str
+    new_item: str
+
+
+class ItemDeleteResponse(BaseModel):
+    message: str
+    deleted_item: str
+    remaining_items_count: int
+
+
+@app.get("/", tags=["Random Playground"])
+async def home():
+    return {"message": "Welcome to the Randomizer API"}
+
+
+@app.get("/random/{max_value}")
+async def get_random_number(max_value: int, tags=["Random Playground"]):
```

Comment:
Consistent with the rest:

```suggestion
@app.get("/random/{max_value}", tags=["Random Playground"])
async def get_random_number(max_value: int):
```